_docs/tasks/traffic-management/ingress.md

```diff
@@ -58,79 +60,16 @@ rules.
      rules:
      - http:
          paths:
-         - path: /.*
+         - path: /status/.*
            backend:
              serviceName: httpbin
              servicePort: 8000
    EOF
    ```
  
    `/.*` is a special Istio notation that is used to indicate a prefix
-   match, specifically a configuration of the form (`prefix: /`). This
-   configuration above will allow access to all URIs in the httpbin
-   service. However we wish to enable access only to specific URIs under the
-   httpbin service. Let us define a default _deny all_ route rule that
-   provides this behavior:
-
-   ```bash
-   cat <<EOF | istioctl create -f -
-   ## Deny all access from istio-ingress
-   apiVersion: config.istio.io/v1alpha2
-   kind: RouteRule
-   metadata:
-     name: deny-route
-   spec:
-     destination:
-       name: httpbin
-     match:
-       # Limit this rule to istio ingress pods only
-       source:
-         name: istio-ingress
-         labels:
-           istio: ingress
-     precedence: 1
-     route:
-     - weight: 100
-     httpFault:
-       abort:
-         percent: 100
-         httpStatus: 403 #Forbidden for all URLs
-   EOF
-   ```   
-2. Now, allow requests to `/status/` prefix by defining a route rule of
-   higher priority.
-
-   ```bash
-   cat <<EOF | istioctl create -f -
-   ## Allow requests to /status prefix
-   apiVersion: config.istio.io/v1alpha2
-   kind: RouteRule
-   metadata:
-     name: status-route
-   spec:
-     destination:
-       name: httpbin
-     match:
-       # Limit this rule to istio ingress pods only
-       source:
-         name: istio-ingress
-         labels:
-           istio: ingress
-       request:
-         headers:
-           uri:
-             prefix: /status
-     precedence: 2 #must be higher precedence than the deny-route
-     route:
-     - weight: 100
-   EOF
-   ```
-  You can use other features of the route rules such as redirects,
-  rewrites, regular expression based match in HTTP headers, websocket
-  upgrades, timeouts, retries, and so on. Please refer to the
-  [routing rules]({{home}}/docs/reference/config/traffic-rules/routing-rules.html)
-  for more details.
-
+   match, specifically a configuration of the form (`prefix: /`).
```

Comment:
specifically a [rule match configuration]({{home}}/docs/reference/config/traffic-rules/routing-rules.html#matchrequest) of the form